library/general/src/lib/ui/widgets.rb

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may find
+# current contact information at www.novell.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm/widget"
+
+module UI
+  module Widgets
+    class KeyboardLayoutTest < CWM::InputField
+      def initialize
+        textdomain "base"
+      end
+
+      def label
+        _("&Test Keyboard Layout")
+      end
+    end
+  end
+end
```

Comment:
I would prefer having this class defined in a separate file (`keyboard_layout_test`). Something like `lib/ui/widgets/keyboard_layout_test`. What do you think?